README.md

```diff
@@ -113,6 +113,49 @@ both served from the `example` folder.
 - `npx react-native start`
 - type `i` or `a` to run on either platform
 
+#### Push Notifications for Android and Firebase services Integration Guide
+
+Push notifications require `react-native-notifications` and Firebase services.
+To integrate with your example app, follow the steps below:
+
+##### 1. Modify MainApplication.java
+
+Add the following imports to `MainApplication.java` file:
```

Comment:
Since the next step is uncomment, should we make this a comment instead of an add?